New operating sounds for wrench, crowbar and wirecutter

TheBoondock opened the pull request:
## About The Pull Request
Medical tools have some operation sounds that added to the aesthetic of their work so I thought why not engineering.

https://github.com/tgstation/tgstation/assets/92416224/30e4558c-8a64-407b-a7fb-b1ad30966fe5





## Why It's Good For The Game
People play engineering to fix stuffs so making the aesthetic of using the tools better should make it fixing more enjoyable.
## Changelog
:cl:

sound: added operating sounds for wrench, wirecutter and crowbar

/:cl:

TheBoondock committed:
add operating sounds for wrench,screwdriver

TheBoondock committed:
new sounds for crowbar, wirecutter as well

Wayland-Smithy commented:
More audio feedback is cool, the wrench sounds nice, but personally am finding the wirecutter and crowbar to an extent a bit tinny and grating on the ears made worse by needing to use them heavily I would imagine.

TheBoondock commented:
Is it because of the pitch or the volume of it?

Wayland-Smithy commented:
Hard to put a finger on it, maybe both for the wirecutter? That's just my singular opinion anyway and I am an audio snob. Could be this is a good candidate for a test merge to get some feedback from the broader player base if the maints have any concerns.

Admiral-Gigaton commented:
crowbar sounds a bit funky but otherwise good. maybe the crowbar could sound more... clunky? heavy?

TheBoondock commented:
> Hard to put a finger on it, maybe both for the wirecutter? That's just my singular opinion anyway and I am an audio snob. Could be this is a good candidate for a test merge to get some feedback from the broader player base if the maints have any concerns.

I am not very good with sound so your input is much needed, should I increase the pitch to make it sounds more clunky and reduce the volume? I do wanna make the crowbar sounds more pronounce.

Wayland-Smithy commented:
The frequency is def the bigger deal. I found a [site](https://academo.org/demos/spectrum-analyzer/) to generate this graph (bottom axis as time) of wirecutter_cut.ogg and compared to all the sample sounds the peaks here are way higher freq.
![image](https://github.com/tgstation/tgstation/assets/64715958/01b31b7f-4b60-45cd-bc7c-508c86d80e21)

ShizCalev commented:
Why is the wirecutter banging against titanium chains

TheBoondock committed:
modify the sound

TheBoondock commented:
https://github.com/tgstation/tgstation/assets/92416224/63d6e212-8325-41af-b35b-d175e359f7ea


https://github.com/tgstation/tgstation/assets/92416224/01120a5a-c9b7-44e2-a713-eff5367a2397

Modified the pitch and volume for both wirecutter and crowbar

Metekillot commented:
> Is it because of the pitch or the volume of it?

It's not the pitch or volume; it's the peaks. Try to smooth them out. The initial pitch and volume were fine, but they peaked too much.

Jacquerel commented:
> crowbar_prying.mp4
> wirecutter_cut.mp4
> 
> Modified the pitch and volume for both wirecutter and crowbar

Is the OP video updated? I would like to hear it in context with the existing sound; the wirecutter one especially seemed way noisier than the noise wirecutters usually make.

TheBoondock commented:
I just figured out how to use limiter in audacity so i limited the wirecutter sound. Updated the pr body to showcase new sound now. Also i added screwdriver sound too but I dont know if theres a screwdriver operation as long as 4 seconds.

TheBoondock committed:
less painful sound and implement cd

TheBoondock committed:
make the wirecutter sounds a little less loud than the initial

ShizCalev commented on code/game/objects/items.dm:
```diff
@@ -1164,6 +1169,18 @@
+		if(islist(operating_sound))
+			played_sound = pick(operating_sound)
+
+		if(TIMER_COOLDOWN_FINISHED(src, COOLDOWN_TOOL_SOUND))
+			playsound(target, played_sound, volume, TRUE)
+			TIMER_COOLDOWN_START(src, COOLDOWN_TOOL_SOUND, 4 SECONDS) //based on our longest sound clip
```
```suggestion
		if(!TIMER_COOLDOWN_FINISHED(src, COOLDOWN_TOOL_SOUND))
			return
		playsound(target, played_sound, volume, TRUE)
		TIMER_COOLDOWN_START(src, COOLDOWN_TOOL_SOUND, 4 SECONDS) //based on our longest sound clip
```

ShizCalev committed:
Update code/game/objects/items.dm

ShizCalev commented:
much better. ♥
